.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: "Release"
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency: release
```

Comment:
This will solve race condition between two PR's and allow only one PR to run at the same time.

It is ok to cancel other pending pipelines if there is new commit since release script will release all changed charts and update index after.

Reply:
I like this. 
:github: rocks 🎸